src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciderConfig.yml

```diff
@@ -0,0 +1,12 @@
+# Order of action priorities
+# Loaded to a map of Decider and action priorities
+# action priorities are given as a list
+# priority order is decreasing in nature.
+---
+deciderActionPriorityOrder:
+  QueueHealthDecider:
+    - ModifyQueueCapacityAction
```

Comment:
We want to use decider and action names (`NAME` field, also returned by `name()`) instead of class names. This lets us refer to these names directly at different places in the code.

Reply:
Gotcha. Updated.